### PR TITLE
Homework 6

### PR DESCRIPTION
**Pull request by valeriiashyshko**

### CODE REVIEW

File: src/homework6/Calculator.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    public static int subtraction(int a, int b) {
+        return a - b;
+    }
+
+    public static int multiplication(int a, int b) {
+        return a * b;
+    }
+
+    public static int division(int a, int b) {
+        if (b != 0)
```
Comment (AndrewSkoba) [inline src/homework6/Calculator.java]: In Java basically if...else statement use with brackets for better readability

File: src/secondPackageForHomework6/CheckCalculator.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package secondPackageForHomework6;
+
+import homework6.Calculator;
+
+public class CheckCalculator {
+    public static void main(String[] args) {
+
+        System.out.println("Addition result is: " + Calculator.addition(3, 15));
```
Comment (AndrewSkoba) [inline src/secondPackageForHomework6/CheckCalculator.java:8]: Can be used static import

File: src/secondPackageForHomework6/CheckUser.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+        user1.makePurchase(44.0);
+        user2.makePurchase(5.58);
+        user3.makePurchase(121.11);
+
+        getUser(user1);
+        getUser(user2);
+        getUser(user3);
+
+    }
```
Comment (AndrewSkoba) [inline src/secondPackageForHomework6/CheckUser.java]: Empty row between methods

File: src/secondPackageForHomework6/CheckCalculator.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
+package secondPackageForHomework6;
```
Comment (AndrewSkoba) [inline src/secondPackageForHomework6/CheckCalculator.java:1]: package name should be like secondpackageforhomework6

File: src/homework6/Calculator.java (hunk 1 above)
Comment (valeriiashyshko) [reply src/homework6/Calculator.java]: Brackets added

File: src/secondPackageForHomework6/CheckCalculator.java (hunk 2 above)
Comment (valeriiashyshko) [reply src/secondPackageForHomework6/CheckCalculator.java:8]: Changed to `import static homework6.Calculator.*; `

File: src/secondPackageForHomework6/CheckUser.java (hunk 3 above)
Comment (valeriiashyshko) [reply src/secondPackageForHomework6/CheckUser.java]: Added an empty row between methods

File: src/secondPackageForHomework6/CheckCalculator.java (hunk 4 above)
Comment (valeriiashyshko) [reply src/secondPackageForHomework6/CheckCalculator.java:1]: Changed the package name to secondpackageforhomework6

**Review by AndrewSkoba — APPROVED** [review]

**Pull request merged**